app/src/main/java/ch/epfl/sdp/appart/utils/UIUtils.java

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.sdp.appart.utils;
+
+import android.util.Log;
+import android.view.View;
+
+import androidx.annotation.StringRes;
+
+import com.google.android.material.snackbar.BaseTransientBottomBar;
+import com.google.android.material.snackbar.Snackbar;
+
+import ch.epfl.sdp.appart.R;
+
+public class UIUtils {
+
+    public static void makeSnakeAndLogOnFail(View view, @StringRes int id, Throwable e) {

```

Comment:
Java doc ? What is the purpose and when should it be used ?

Reply:
This was used to solve the duplicate code issue from CodeClimate. It is used in several acitvity to make a message appear. I agree it lacks documentation.